added option for alternate (alias) domain, added `docker-compose.yml`…

chrismazanec opened the pull request:
…, formatting, docs

Commit:
added option for alternate (alias) domain, added `docker-compose.yml`, formatting, docs

maksimstojkovic commented:
Hi @chrismazanec,

Thanks for your contribution!
I'll have a look at your proposed changes at some point over the next week or two, and get back to you.

From an initial glance, it looks promising.

maksimstojkovic committed:
alternative domain merge prep

Added note about proxied DNS records to README.md
Updated CNAME record docs
Adjusted format of docker-compose.yml variables
Re-ordered README.md variables to match docker-compose.yml
Added DNS record conditions to README.md
Added duckdns container to docker-compose.yml
Added default value checks to start.sh

maksimstojkovic commented:
Hi @chrismazanec,

I reviewed your changes and they work! 🎉

I made some minor modifications to the documentation and docker-compose file to better explain the usage of this feature, but otherwise all of your changes were spot on.

Great job! 👏

chrismazanec commented:
thanks a lot for merging and adjustments ;)
